pages/common/ern.md

```diff
@@ -0,0 +1,36 @@
+# ern
+
+> Electrode Native is a mobile platform that integrates React Native components into mobile applications.
+> More information: <https://www.electrode.io/site/native.html>.
+
+- Create a new ern application(miniapp):
```

Comment:
```suggestion
- Create a new ern application (MiniApp):
```

Reply:
```suggestion
- Create a new `ern` application (`MiniApp`):
```

Reply:
Good suggestions, will make the changes.